path-of-titansmetaconfig.json

```diff
@@ -1,4 +1,16 @@
 [
+    {    
+        "ConfigFile":"PathOfTitans/Saved/Config/WindowsServer/Game.ini",
+        "AutoMap":true,
+        "ConfigType":"ini",
+		"ConfigFormatRegex":"^(?<key>.+?)=(?<value>.*?)$"
```

Comment:
Very minor, but the line indentations are misaligned on this line for both files.